### PR TITLE
Fixed tiny help icon

### PR DESCRIPTION
**Pull request by Emmanueluko3**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+REACT_APP_MONGODB_READ_TOKEN=D5Kk0zuirvDkz9XpXBt71rN2sRSiU4CeX5CLKJIef7PSWgIKifGZ8IG03fCdxYef
```
Comment (dankelleher) [inline .env:1]: Please remove this file, the .env file should be in the /app folder and not committed

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,4 @@
+.env
```
Comment (dankelleher) [inline .gitignore:1]: this line can be removed - there is no top-level .env file

**Review by dankelleher — CHANGES_REQUESTED** [review]

File: .env (hunk 1 above)
Comment (Emmanueluko3) [reply .env:1]: Ok

File: .gitignore (hunk 2 above)
Comment (Emmanueluko3) [reply .gitignore:1]: Yeah, I already removed it

**Pull request closed without merging**